packages/create-svelte/bin.js

```diff
@@ -134,7 +134,7 @@ async function main() {
  */
 function write_template_files(template, typescript, name, cwd) {
 	const dir = dist(`templates/${template}`);
-	copy(`${dir}/assets`, cwd, (name) => name.replace('gitignore', '.gitignore'));
+	copy(`${dir}/assets`, cwd, (name) => name.replace('DOT-', '.'));
```

Comment:
Maybe use regex here too, for the off chance that there will ever be any file containing `DOT-` in it's name...
```suggestion
	copy(`${dir}/assets`, cwd, (name) => name.replace(/^DOT-/, '.'));
```

Reply:
`DOT-` could also be extracted as constsant into `packages/create-svelte/utils.js`